test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Verb.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+﻿Describe "Get-Verb" -Tags "CI" {
+
+    It "Should get a list of Verbs" {
+        Get-Verb | Should not be $null
+    }
+
+    It "Should get a specific verb" {
+        (Get-Verb -Verb Add | Measure-Object).Count | Should be 1
+        (Get-Verb -Verb Add -Group Common | Measure-Object).Count | Should be 1
+    }
+
+    It "Should get a specific group" {
+        Get-Verb -Group Common | Should not be $null
```

Comment:
>-Verb -Group Common | Should not be $null [](start = 11, length = 41)

this doesn't actually validate that you got the proper group
i think you're looking for this:
`(Get-Verb -Group Common).Group | Sort-Object -uniq | Should be Common`